stdlib/task/src/main/ballerina/src/task/scheduler.bal

```diff
@@ -21,17 +21,17 @@ public type Scheduler object {
     # Initializes a `task:Scheduler` object. This may panic if the initialization causes any error due to
     # a configuration error.
     #
-    # + configurations - A `task:TimerConfiguration` or `task:AppointmentConfiguration` record to define the
-    # `task:Sceduler` behavior
-    public function __init(TimerConfiguration|AppointmentConfiguration configurations) {
-        self.taskListener = new(configurations);
+    # + configuration - `task:TimerConfiguration` or `task:AppointmentConfiguration` record to define the
+    #                   `task:Sceduler` behavior
+    public function __init(TimerConfiguration|AppointmentConfiguration configuration) {
+        self.taskListener = new(configuration);
     }
 
     # Attaches the provided `service` to the task.
     #
-    # + serviceToAttach - Ballerina `service` object which needs to be attached to the task.
-    # + attachments - Set of optional parameters which needs to passed inside the resources.
-    # + return - Returns `task:SchedulerError` if the process failed due to any reason, nil otherwise.
+    # + serviceToAttach - Ballerina `service` object which needs to be attached to the task
+    # + attachments - Set of optional parameters which needs to passed inside the resources
+    # + return - `task:SchedulerError` if the process failed due to any reason, nil otherwise
```

Comment:
```suggestion
    # + return - `task:SchedulerError` if the process failed due to any reason, nil otherwise
```
Same as above. Also, please refer the template for the correct format of the return statement description.

Reply:
Fixed in all places